custom_components/ramses_cc/__init__.py

```diff
@@ -109,7 +109,7 @@ class RamsesController:
 
     _port_name: str
     _client_config: dict
-    _client: Gateway
+    _client: Gateway | None
```

Comment:
if this is never None, other than at instantiation, and never subsequently tested for None-ness, if woudl leave it as is, and set a mypy type hint

Reply:
The client is only created later in `_create_client`. It's tested for None-ness in `_async_create_update_tasks` so that we only create the update tasks once all platforms have registered *and* we have created a client.